content/courses/program-optimization/program-configuration.md

```diff
@@ -866,51 +859,50 @@ pub fn payment_handler(ctx: Context<Payment>, amount: u64) -> Result<()> {
 
 #### 10. Test
 
-Now that we're done implementing our new program configuration struct and
+Now that we're done implementing our new admin configuration struct and
 instructions, let's move on to testing our updated program. To begin, add the
-PDA for the program config account to the test file.
+PDA for the admin config account to the test file.
 
 ```typescript
 describe("config", () => {
   ...
-  const programConfig = findProgramAddressSync(
-    [Buffer.from("program_config")],
+  const adminConfig = PublicKey.findProgramAddressSync(
+    [Buffer.from("admin_config")],
     program.programId
-  )[0]
+  )[0];
 ...
 ```
 
 Next, update the test file with three more tests testing that:
 
-1. The program config account is initialized correctly
+1. The admin config account is initialized correctly
 2. The payment instruction is functioning as intended
 3. The config account can be updated successfully by the admin
 4. The config account cannot be updated by someone other than the admin
 
-The first test initializes the program config account and verifies that the
-correct fee is set and that the correct admin is stored on the program config
+The first test initializes the admin config account and verifies that the
+correct fee is set and that the correct admin is stored on the admin config
 account.
 
 ```typescript
-it("Initialize Program Config Account", async () => {
+it("Initialize Admin config should be successfully", async () => {
   const tx = await program.methods
-    .initializeProgramConfig()
+    .initializeAdminConfig()
     .accounts({
-      programConfig: programConfig,
       feeDestination: feeDestination,
       authority: wallet.publicKey,
-      systemProgram: anchor.web3.SystemProgram.programId,
+      programData: programDataAddress,
     })
     .rpc();
 
   assert.strictEqual(
     (
-      await program.account.programConfig.fetch(programConfig)
+      await program.account.adminConfig.fetch(adminConfig)
     ).feeBasisPoints.toNumber(),
     100,
   );
   assert.strictEqual(
-    (await program.account.programConfig.fetch(programConfig)).admin.toString(),
+    (await program.account.adminConfig.fetch(adminConfig)).admin.toString(),
```

Comment:
For these strictEquals, it's a little busy, just add a variable name rather than the inline await.

```
const adminConfig = await program.account.adminConfig.fetch(adminConfig);
assert.strictEqual(adminConfig.admin, wallet.publicKey.toString())
```